py2many/rewriters.py

```diff
@@ -11,7 +11,9 @@
 
 class InferredAnnAssignRewriter(ast.NodeTransformer):
     def visit_Assign(self, node):
-        target = node.targets[0]
+        target = node.targets[0]  # Assumes all targets have same annotation
```

Comment:
there are still many other uses of `node.targets[0]` which are likely are other bugs like https://github.com/adsharma/py2many/issues/362 .  We should check them all, and add a comment like this if they are safe, so we can easily check for any new cases of this type of bug.